src/Apps/FeatureAKG/routes.tsx

```diff
@@ -0,0 +1,46 @@
+import loadable from "@loadable/component"
+import { RouteConfig } from "found"
+import { graphql } from "react-relay"
+
+export const routes: RouteConfig[] = [
+  {
+    path: "/campaign/art-keeps-going",
+    getComponent: () => loadable(() => import("./FeatureApp")),
```

Comment:
This only applies in the new app shell but we'll want to preload this JS bundle in parallel with the relay req. This prevents the slight flash post route transition as the bundle split loads:

```jsx
const FeatureApp = loadable(() => import("./FeatureApp"))
...
prepare: () => {
  FeatureApp.preload()
},
```

Here's where [it's called](https://github.com/artsy/reaction/blob/master/src/Apps/Components/AppShell.tsx#L21-L34).